stable/datadog/templates/containers-common-env.yaml

```diff
@@ -53,17 +53,16 @@
 - name: {{ $value.name }}
   value: {{ $value.value | quote }}
 {{- end }}
-{{- if .Values.datadog.dockerSocketPath }}
+{{- if .Values.datadog.criSocketPath }}
+- name: DD_CRI_SOCKET_PATH
+  value: {{ print "/host/" .Values.datadog.criSocketPath | clean }}
+{{- else }}
 - name: DOCKER_HOST
 {{- if eq .Values.targetSystem "linux" }}
-  value: unix://{{ .Values.datadog.dockerSocketPath }}
+  value: unix://{{ print "/host/" (include "datadog.dockerOrCriSocketPath" .) | clean }}
 {{- end }}
 {{- if eq .Values.targetSystem "windows" }}
-  value: npipe://{{ .Values.datadog.dockerSocketPath | replace "\\" "/" }}
-{{- end }}
+  value: npipe://{{ print "/host/" (include "datadog.dockerOrCriSocketPath" .) | clean | replace "\\" "/" }}
```

Comment:
It'll never be a folder on Windows, you should keep:
`value: npipe://{{ .Values.datadog.dockerOrCriSocketPath | replace "\\" "/" }}`

Reply:
Change reverted for Windows to that it applies to Linux only.